### DISCUSSION TITLE
[Weak UI/UX] - When starting a discussion, 'Start Discussion' disabled, no obvious reason why

### DISCUSSION THREAD

**Discussion by YoloClin** [community/community#16554] [Discussions]
To reproduce:

- Open https://github.com/github/feedback/discussions/new
- Type a title, and body message
- Click 'Start discussion', it's disabled so you can't, it doesn't tell you why.
- Inspect Element, remove 'disabled' from 'Start Discussion' button, click again -> 'You must enter Category'

This caught me, I found myself hacking HTML to understand why I couldn't start a discussion. I didn't see the category drop-down, which is user error but something that could be avoided. A better UX would be a hover message or to let the user click the button and tell them _why_, or just to remove the disabled flag and let JS validation execute (this looks like it might have behaved like that at some point).

**Comment by YoloClin** [comment]
For those playing at home this is unrelated to my original post, there's no way to communicate on the referenced CVE. 

@Gauravp-NEC From memory I tried 2.2.x and 2.10.0, the relevant logic for code exec traces back to 2.2.x, there are other issues related to the API that mean that not only is it RCE, it's also (for unrelated reasons) a DoS and Dataset disclosure weakness. CKAN no longer support 2.7.x as far as I can tell, so I would suggest patching regardless.

**Reply by Gauravp-NEC** [reply]
@YoloClin , Thank you quick response. I apologies for the unrelated post here. I am deleting the post.
